webhook_features/src/features/summary_comment.rs

```diff
@@ -326,74 +320,22 @@ struct AckCommit {
 
 fn parse_review(comment: &str) -> Option<AckCommit> {
     let comment = comment.to_lowercase();
```

Comment:
Any reason to lowercase this, given that this is against the guideline https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review and won't be picked up by the other script: https://github.com/bitcoin-core/bitcoin-maintainer-tools/blob/78ab16ae88af7a5ef886ae8cef6df2e9ef3f6085/github-merge.py#L214

Reply:
I've seen many instances of `code review ACK` or `Code review ACK` so I think it's just easier to leave the `to_lowercase`.

Reply:
You can write whatever you want before `ACK`, this won't be affected by the case. See the passing test:


```diff
diff --git a/webhook_features/src/features/summary_comment.rs b/webhook_features/src/features/summary_comment.rs
index 751b114b3..341974bf2 100644
--- a/webhook_features/src/features/summary_comment.rs
+++ b/webhook_features/src/features/summary_comment.rs
@@ -504,7 +504,7 @@ mod tests {
                 expected: None,
             },
             TestCase {
-                comment: "crACK 1234567890123456789012345678901234567890",
+                comment: "cOde rrrr ACK 1234567890123456789012345678901234567890",
                 expected: Some(AckCommit {
                     ack_type: AckType::Ack,
                     commit: Some("1234567890123456789012345678901234567890".to_string()),

Reply:
I'm not sure what you mean. The regex that matches this string is:
`(r".*\b(?:re)?(ack|utack|tack)((\s)*[0-9a-f]{6,40})\b.*", AckType::Ack),`

This is like `I agree with the changes, ACK 12345678`, the ACK should still be detected right?

Reply:
Yes, it should and will be detected. There is no need calling `to_lowercase`

Reply:
Oh right my bad

Reply:
Dropped `Code Review ACK` and `Concept NACK` patterns as they are both implicitly matched and updated case of regexes.

https://github.com/MarcoFalke/DrahtBot/compare/104153e5fe471ff6df52558a9784e9c1b9fa24f0..0f1ed3d70e3b45784b52308d7326eceb492e9aa5